packages/drivers/odsp-driver/src/test/localSnapshots/localSnapshot1.json

```diff
@@ -0,0 +1 @@
thor\":\"Microsoft and contributors\",\"main\":\"dist/index.js\",\"module\":\"lib/index.js\",\"types\":\"dist/index.d.ts\",\"scripts\":{\"build\":\"concurrently npm:build:compile npm:lint\",\"build:compile\":\"concurrently npm:tsc npm:build:esnext\",\"build:esnext\":\"tsc --project ./tsconfig.esnext.json\",\"build:full\":\"concurrently npm:build npm:webpack\",\"build:full:compile\":\"concurrently npm:build:compile npm:webpack\",\"clean\":\"rimraf dist lib *.tsbuildinfo *.build.log\",\"eslint\":\"eslint --format stylish src\",\"eslint:fix\":\"eslint --format stylish src --fix --fix-type problem,suggestion,layout\",\"lint\":\"npm run eslint\",\"lint:fix\":\"npm run eslint:fix\",\"prepack\":\"npm run webpack\",\"start\":\"webpack serve --config webpack.config.js\",\"start:docker\":\"webpack serve --config webpack.config.js --env mode=docker\",\"start:r11s\":\"webpack serve --config webpack.config.js --env mode=r11s\",\"start:spo\":\"webpack serve --config webpack.config.js --env mode=spo\",\"start:spo-df\":\"webpack serve --config webpack.config.js --env mode=spo-df\",\"start:tinylicious\":\"webpack serve --config webpack.config.js --env mode=tinylicious\",\"test\":\"npm run test:jest\",\"test:jest\":\"jest\",\"test:jest:verbose\":\"cross-env FLUID_TEST_VERBOSE=1 jest\",\"tsc\":\"tsc\",\"tsfmt\":\"tsfmt --verify\",\"tsfmt:fix\":\"tsfmt --replace\",\"webpack\":\"webpack --env production\",\"webpack:dev\":\"webpack --env development\"},\"dependencies\":{\"@fluid-example/example-utils\":\"^2.0.0\",\"@fluidframework/aqueduct\":\"^2.0.0\",\"@fluidframework/common-definitions\":\"^0.20.1\",\"@fluidframework/core-interfaces\":\"^2.0.0\",\"@fluidframework/ink\":\"^2.0.0\",\"react\":\"^16.10.2\"},\"devDependencies\":{\"@fluid-tools/webpack-fluid-loader\":\"^2.0.0\",\"@fluidframework/build-common\":\"^0.23.0\",\"@fluidframework/eslint-config-fluid\":\"^0.28.2000\",\"@fluidframework/test-tools\":\"^0.2.3074\",\"@rushstack/eslint-config\":\"^2.5.1\",\"@types/expect-puppeteer\":\"2.2.1\",\"@types/jest\":\"22.2.3\",\"@types/jest-environment-puppeteer\":\"2.2.0\",\"@types/node\":\"^14.18.0\",\"@types/puppeteer\":\"1.3.0\",\"@types/react\":\"^16.9.15\",\"concurrently\":\"^6.2.0\",\"cross-env\":\"^7.0.2\",\"css-loader\":\"^1.0.0\",\"eslint\":\"~8.6.0\",\"jest\":\"^26.6.3\",\"jest-junit\":\"^10.0.0\",\"jest-puppeteer\":\"^4.3.0\",\"less\":\"~3.9.0\",\"less-loader\":\"^4.1.0\",\"puppeteer\":\"^1.20.0\",\"rimraf\":\"^2.6.2\",\"style-loader\":\"^1.0.0\",\"ts-loader\":\"^9.3.0\",\"typescript\":\"~4.5.5\",\"typescript-formatter\":\"7.1.0\",\"url-loader\":\"^2.1.0\",\"webpack\":\"^5.72.0\",\"webpack-cli\":\"^4.9.2\",\"webpack-dev-server\":\"~4.6.0\",\"webpack-merge\":\"^5.8.0\"},\"fluid\":{\"browser\":{\"umd\":{\"files\":[\"dist/main.bundle.js\"],\"library\":\"main\"}}},\"jest-junit\":{\"outputDirectory\":\"nyc\",\"outputName\":\"jest-junit-report.xml\"}},\"config\":{}},\"approvalSequenceNumber\":0,\"commitSequenceNumber\":0,\"sequenceNumber\":0}]]","size":3000}],"ops":[]}
```

Comment:
Not sure whether it will appear like this after submitting your PR: https://github.com/microsoft/FluidFrameworkTestData/blob/623cd9a155435a59654b4101068b99ec397fd1dc/snapshotTestContent/Authoring-MVP3-(created-Aug-8)/current_snapshots/snapshot_1000_0.json
If not, I think we can beautify it in this PR so that it does not appear in single line.